### PR TITLE
Fixing other message filtering issue in log error by class gadget

### PR DESCRIPTION
**Pull request by ksdperera**
Updating log filtering gadget accordingly log error by class fixes
Reformatting main.js

### CODE REVIEW

File: log-analyzer/features/org.wso2.analytics.common.la.apim.feature/src/main/resources/gadgets/LogErrorFilteredMessage/js/main.js [hunk 1]
```diff
@@ -15,15 +15,17 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-var gatewayPort = location.port -9443 + 8243; //Calculate the port offset based gateway port.
-var serverUrl = "https://"+location.hostname +":"+ gatewayPort+"/LogAnalyzerRestApi/1.0";
+var gatewayPort = location.port - 9443 + 8243; //Calculate the port offset based gateway port.
```
Comment (keizer619) [inline log-analyzer/features/org.wso2.analytics.common.la.apim.feature/src/main/resources/gadgets/LogErrorFilteredMessage/js/main.js:18]: @ksdperera In a lot of places pf log analyser code this calculation has been done. Can't we implement this in a common util JS file and reuse it? else if there are improvements then we need to change everywhere.

File: log-analyzer/features/org.wso2.analytics.common.la.apim.feature/src/main/resources/gadgets/LogErrorMessage/js/main.js [hunk 2]
```diff
@@ -134,56 +134,56 @@ function fetch() {
...
                 newFrom.setHours(0);
                 newFrom.setMinutes(0);
                 newFrom.setSeconds(0);
                 newTo.setHours(0);
                 newTo.setMinutes(0);
                 newTo.setSeconds(0);
-                while(!(newFrom.getTime() >= newTo.getTime())){
-                    mockData.push([newFrom.toDateString(),0,"No entries","No entries",0]);
-                    newFrom.setHours(newFrom.getHours()+24);
+                while (!(newFrom.getTime() >= newTo.getTime())) {
```
Comment (keizer619) [inline log-analyzer/features/org.wso2.analytics.common.la.apim.feature/src/main/resources/gadgets/LogErrorMessage/js/main.js:144]: @ksdperera we can just simple write like this. 
while (newFrom.getTime() < newTo.getTime())

File: log-analyzer/features/org.wso2.analytics.common.la.apim.feature/src/main/resources/gadgets/LogErrorFilteredMessage/js/main.js (hunk 1 above)
Comment (ksdperera) [reply log-analyzer/features/org.wso2.analytics.common.la.apim.feature/src/main/resources/gadgets/LogErrorFilteredMessage/js/main.js:18]: Hi Tharik,

This is fixed by the commits in : https://github.com/wso2/shared-analytics/pull/59

Thanks,
SajithD

File: log-analyzer/features/org.wso2.analytics.common.la.apim.feature/src/main/resources/gadgets/LogErrorMessage/js/main.js (hunk 2 above)
Comment (ksdperera) [reply log-analyzer/features/org.wso2.analytics.common.la.apim.feature/src/main/resources/gadgets/LogErrorMessage/js/main.js:144]: Hi Tharik,

This is fixed by the commits in : https://github.com/wso2/shared-analytics/pull/90

Thanks,
SajithD

**Pull request merged**